share/xtp/xml/gencube.xml

```diff
@@ -1,7 +1,5 @@
 <options>
 <gencube help="Tool to generate cube files from .orb file" section="sec:gencube">
-        <output help="Output file" default="votca.cube">votca.cube</output>
-        <input help="Input file" default="votca.orb">votca.orb</input>
```

Comment:
So I cannot specify the output manually even if I want to?

Reply:
of course, you can change the name of the input/output files,  see: https://github.com/votca/xtp/blob/name_xtp_tools/src/libxtp/tools/gencube.cc#L39
the `-n name` option is there just to help the user to avoid having to fill all the names in the XML files manually.